arches_lingo/src/arches_lingo/components/generic/LabelEditor.vue

```diff
@@ -0,0 +1,141 @@
+<script setup lang="ts">
+import { defineProps, onMounted, ref } from "vue";
+import { fetchControlledListOptions } from "@/arches_lingo/api.ts";
+
+import NonLocalizedString from "@/arches_lingo/components/generic/NonLocalizedString.vue";
+import ReferenceDatatype from "@/arches_lingo/components/generic/ReferenceDatatype.vue";
+// import ResourceInstanceRelationships from "@/arches_lingo/components/generic/ResourceInstanceRelationships.vue";
```

Comment:
nit: remove this instead of commenting it out?

Reply:
Two resource instance nodes in this nodegroup, just hadn't stubbed those out - they're now and import is used 👍🏼